react/features/connection-indicator/components/ConnectionIndicator.js

```diff
@@ -6,44 +7,42 @@ import { ConnectionStatsTable } from '../../connection-stats';
 
 import statsEmitter from '../statsEmitter';
 
-declare var $: Object;
 declare var interfaceConfig: Object;
 
-// Converts the percent for connection quality into a string recognized for CSS.
-const QUALITY_TO_WIDTH = [
-
-    // Full (5 bars)
-    {
-        percent: 80,
-        width: '100%'
-    },
-
-    // 4 bars
-    {
-        percent: 60,
-        width: '80%'
-    },
-
-    // 3 bars
-    {
-        percent: 40,
-        width: '55%'
-    },
-
-    // 2 bars
-    {
-        percent: 20,
-        width: '40%'
-    },
-
-    // 1 bar
-    {
-        percent: 0,
-        width: '20%'
-    }
+/**
+ * The connection quality percentage that must be reached to be considered
+ * of good quality and can result in the connection indicator being hidden.
+ *
+ * @type {number}
+ */
+const INDICATOR_DISPLAY_THRESHOLD = 64;
 
-    // Note: we never show 0 bars.
-];
+/**
+ * A map of connection quality percentages and corresponding colors to display
+ * when the current connection quality meets or exceeds the percentage.
+ *
+ * @type {Object}
+ */
+const CONNECTION_TO_COLOR_MAP = {
```

Comment:
I didn't know what colors to use so I just picked some from a red-to-green scale. I also didn't know at what percentages to trigger different behaviors, so I copied the original implementation--https://github.com/jitsi/jitsi-meet/pull/1012.